pkg/virt-handler/vm.go

```diff
@@ -222,11 +223,15 @@ func NewVirtualMachineController(
 		permissions = "rwm"
 	}
 
+	// Get the hypervisor device name from cluster config
+	hypervisorName := clusterConfig.GetConfig().HypervisorConfiguration.Name
+	hypervisor := hypervisor.NewHypervisor(hypervisorName)
```

Comment:
This seems to be extra hoop that is not needed, I would just do 
```
deviceManager.PermanentHostDevicePlugins(clusterConfig.GetConfig().HypervisorConfiguration.Name maxDevices, permissions),
```
for now

Reply:
It would be better to keep the name of the hypervisor config independent of the device exposed by kernel. So I'd like to keep using the `GetDevice()` function.